src/jit/simdcodegenxarch.cpp

```diff
@@ -475,10 +476,10 @@ instruction CodeGen::getOpForSIMDIntrinsic(SIMDIntrinsicID intrinsicId, var_type
 // Notes:
 //    This is currently only supported for floating point types.
 //
-void CodeGen::genSIMDScalarMove(var_types type, regNumber targetReg, regNumber srcReg, SIMDScalarMoveType moveType)
+void CodeGen::genSIMDScalarMove(
+    var_types targetType, var_types baseType, regNumber targetReg, regNumber srcReg, SIMDScalarMoveType moveType)
```

Comment:
However, I do see the targetType is been used in this method at  "case SMT_ZeroInitUpper:" (when it's SSE, not within FEATURE_AVX_SUPPORT define) when srcReg != targetReg, it is been used in genSIMDZero(targetType, TYP_FLOAT, targetReg);